public/pages/ManagedIndices/utils/constants.ts

```diff
@@ -32,4 +32,6 @@ export const ACTIONS: Actions = {
   close: "Close",
   read_only: "Read Only",
   read_write: "Read Write",
+  replica_count: "Replica Count",
```

Comment:
Missing constant for ```force_merge```

Reply:
Added in a different PR:
https://github.com/opendistro-for-elasticsearch/index-management-kibana-plugin/pull/46